.changes/next-release/bugfix-socialmessaging-6243.json

```diff
@@ -0,0 +1,5 @@
+{
+  "type": "bugfix",
+  "category": "socialmessaging",
+  "description": "Fixes incorrect customization for renaming a socialmessaging command"
```

Comment:
I'd describe what is changing from a user's perspective, since "customization for renaming" is internal.